src/sys/unix/selector/epoll.rs

```diff
@@ -121,6 +121,7 @@ pub type Events = Vec<Event>;
 pub mod event {
     use crate::sys::Event;
     use crate::Token;
+    use std::fmt::Write as FmtWrite;
```

Comment:
You're just import the trait right, so there is no need for the `as FmtWrite`.

Reply:
Same for the other places.

Reply:
ok, I removed "as FmtWrite"